README.md

```diff
@@ -80,32 +177,35 @@ It takes three arguments:
 
 If you are creating a use case for a component, you *must* namespace your use case name with the name of your component.
 
+### Markup
+
+When using the build service or importing the module with [silent mode](http://origami.ft.com/docs/syntax/scss/#silent-styles) set to false, o-colors provides you with helper classes to access the color palette. All palette colors are available as `.o-colors-palette-[NAME]` (which style just `background-color`) and use cases are available as `.o-colors-[USECASE]-[PROPERTY]` (which style the appropriate property):
+
+```html
+<p class="o-colors-body-text">Article text</p>
+```
+
+### CSS Variables
+
+When using the build service or importing the module with silent mode set to false, o-colors will output all the palette colors as [CSS Variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables). These will use the format `--o-colors-{name}` (e.g. `--o-colors-black` and `--o-colors-teal`).
+
 
-## Development
+## Migration guide
 
-Instructions for maintaining the color palette for non-developers:
+### Upgrading from v3.x.x to v4.x.x
 
-### Getting started
+o-colors v4.x.x updates the entire palette of colors and adds a lot more functionality through new mixins and functions. The palette has been reduced from over 60 colors to a base palette of around 20 colors that can be manipulated through the new functionality in v4.x.x.
+
+To migrate from v3.x.x to use v4.x.x you will need to update the palette colors you are requesting using `oColorsFor`, `oColorsSetUseCase`, and `oColorsGetPaletteColor`. To work out which color names you need to update, we've created a [table showing which colors should now be used in place of the old v3.x.x palette colors](migration-guide.md).
+
+The use cases have also been updated to add new use cases for opinion, hero, and highlight branding, but also removes support for the `product-brand` use case.
+
```

Comment:
[restructure suggestion]
New use cases have been added for `opinion`, `hero` and `highlight` branding.
The `product-brand` use case has been removed.